hnn_core/viz.py

```diff
@@ -777,6 +775,67 @@ def plot_cell_morphology(cell, ax, show=True):
     ax.set_xlim((cell.pos[1] - 250, cell.pos[1] + 150))
     ax.set_zlim((cell.pos[2] - 100, cell.pos[2] + 1200))
 
+    ax = _plot_cell(cell, ax=ax, plt=plt, show=True)
+
+    plt.tight_layout()
+    plt_show(show)
+    return ax
+
+def plot_cell_morphologies(net, ax=None, show=True):
+    """Plot the morphology of the network cells
+
+    Parameters
+    ----------
+    net : instance of Network
+        The network object
+    ax : instance of matplotlib Axes3D | None
+        An axis object from matplotlib. If none,
+        a new figure is created.
+    Show : bool
+        If True, show the figure
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        The matplotlib figure handle
+    """
+
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D  # noqa: F401 unused import
+
+    if ax is None:
+        plt.figure()
+        ax = plt.axes(projection='3d')
+
+    colors = ['b', 'c', 'r', 'm']
```

Comment:
I would set the colormap (e.g., [`'cividis'`](https://matplotlib.org/stable/tutorials/colors/colormaps.html)) as a default arg when calling the function.